### PR TITLE
Update module-listing.de.md

### PR DESCRIPTION
**Pull request by Tastaturberuf**
Fix PHP warnings
Fix PHP opening tag

### CODE REVIEW

File: docs/manual/guides/module-listing.de.md [hunk 1]
```diff
@@ -320,15 +320,16 @@ Das bisherige Template »list_default_member.html5« ergänzen wir wie folgt:
...
 	<?php foreach ($this->tbody as $class => $row): ?>
 		<div class="block memberitem <?= $class ?>"><p>
-		  <a href="mailto:<?= $row['email']['raw'] ?>">
-		  <?= $row['firstname']['content'] ?> <?= $row['lastname']['content'] ?></a>
-		  <span><?= $row['street']['content'] ?> - 
-		  <?= $row['postal']['content'] ?> <?= $row['city']['content'] ?></span>
+		  <a href="mailto:<?= $row['email']['raw'] ?? ''?>">
+		  <?= $row['firstname']['content'] ?? ''?> <?= $row['lastname']['content'] ?></a>
+		  <span><?= $row['street']['content'] ?? '' ?> - 
+		  <?= $row['postal']['content'] ?? '' ?> <?= $row['city']['content'] ?? ''?></span>
```
Comment (fritzmg) [inline docs/manual/guides/module-listing.de.md]: These are not nullable fields in the database, can they ever be null here?

**Comment by Tastaturberuf** [comment]
If the string in backend is empty the key would not exist. Seems a bug to me.

File: docs/manual/guides/module-listing.de.md (hunk 1 above)
Comment (Tastaturberuf) [reply docs/manual/guides/module-listing.de.md:329]: If the string in backend is empty the key would not exist. Seems a bug to me.

Comment (Tastaturberuf) [reply docs/manual/guides/module-listing.de.md]: youre are right, my client has forgotten to select `postal` in den module. I get a warning that the array key dosnt exist.

**Review by fritzmg — APPROVED** [review]

**Comment by Tastaturberuf** [comment]
ah forgot the english version. thx for fixing

**Review by netzarbeiter — APPROVED** [review]
Thank you, Daniel

**Pull request merged**